### PR TITLE
created index page

### PR DESCRIPTION
**Pull request by lucianmot**
chore: add index page
test: test functionality of buttons on index page

### CODE REVIEW

File: spec/features/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  scenario "when you visit '/' you should be able to click sign up" do
+    visit "/"
+    click_button "Sign up"
+    expect(current_path).not_to eq("/")
+    expect(current_path).to eq("/users/sign_up")
+  end
+
+  scenario "when you visit '/' you should be able to click log in" do
```
Comment (Danielandro) [inline spec/features/index_spec.rb]: should we use sign-in in the description, as you've used it within the test

File: spec/features/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    visit "/"
+    click_button "Sign in"
+    expect(current_path).not_to eq("/")
```
Comment (ajmpawlik) [inline spec/features/index_spec.rb]: Nice check if we are not on the same page anymore.

File: app/views/root/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<h1> Welcome to Acebook </h1>
+
+
+<form action="/users/sign_up" method="get" id="Sign up">
```
Comment (Danielandro) [inline app/views/root/index.html.erb]: should this be a post request, as we're sending form data?

File: spec/features/index_spec.rb (hunk 1 above)
Comment (ajmpawlik) [reply spec/features/index_spec.rb]: Well spotted Danny! I second that.

**Pull request merged**